tests/Unit/Client/UpdaterTest.php

```diff
@@ -40,20 +41,15 @@ public function testCheckRollbackAttackNoAttack(): void
 
         // The incoming version is newer than the local version, so no
         // rollback attack is present.
-        $localMetadata = $this->getMockBuilder(MetadataBase::class)->disableOriginalConstructor()->getMock();
-        $localMetadata->expects(self::any())->method('getType')->willReturn('any');
-        $localMetadata->expects(self::any())->method('getVersion')->willReturn(1);
+        $localMetadata = $this->mockMetadataFromJson('{"signed": {"_type": "any", "version": 1}}');
```

Comment:
This test should probably now be in `MetadataBaseTest` with overrides in `TimestampMetadataTest`  and `SnapshotMetadataTest`